modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/parser/CmmnParserImpl.java

```diff
@@ -37,7 +37,7 @@
  */
 public class CmmnParserImpl implements CmmnParser {
 
-    private final Logger logger = LoggerFactory.getLogger(CmmnParserImpl.class);
+    private final Logger LOGGER = LoggerFactory.getLogger(CmmnParserImpl.class);
```

Comment:
This is an instance field, not a `static final` one. The logger should stay lowercase here.

Reply:
Done